index.html

```diff
@@ -180,5 +180,37 @@ <h2 class="about-me-heading">About Me</h2>
      </ul>
      </div>
     </section> 
+    <!-- Form -->
+    <section class="contact-form">
+    <h2>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h2>
+    <form action="https://formspree.io/f/mjvjlpvw"    method="post">
+        <ul>
+            <li>
+               
+                <input type="text" id="name" name="user_name" placeholder="Full name" maxlength="20" required>
+            </li>
+            <li>
+               
+               <input type="email" id="mail" name="user_email" placeholder="Email addresss" required>
+           </li>
+           <li>
+               
+              <textarea name="user_msg" id="msg" cols="30" rows="10" placeholder="Enter text here" maxlength="500" required></textarea>
+           </li>
+        </ul>
+        <button type="submit" >Get in touch</button>
+    </form>
```

Comment:
- [ ] Name input should have maxlength="30" attribute. Kindly update the current validation.

Reply:
I just made it correct.. Thanks for your review.